script.js

```diff
@@ -0,0 +1,119 @@
+const clock = document.querySelector("h1#clock");
+const addButton = document.getElementById("add-button");
+const toDoForm = document.getElementById("todo-form");
+const toDoInput = toDoForm.querySelector("input");
+const toDoList = document.getElementById("todo-list");
+const TODOS_KEY = "todos";
+const savedToDos = localStorage.getItem(TODOS_KEY);
+const doneList = document.getElementById("done-list");
+const DONETODOS_KEY = "donetodos";
+const savedDoneToDos = localStorage.getItem(DONETODOS_KEY);
+let toDos = [];
+let doneToDos = [];
+addButton.addEventListener("click", handleAddButtonClick);
+toDoForm.addEventListener("submit", handleToDoSubmit);
+function getClock() {
+  const date = new Date();
+  const hours = String(date.getHours()).padStart(2, "0");
+  const minutes = String(date.getMinutes()).padStart(2, "0");
+  const seconds = String(date.getSeconds()).padStart(2, "0");
+  const amPm = hours >= 12 ? "PM" : "AM";
+  clock.innerText = `${hours}:${minutes}:${seconds} ${amPm}`;
+  clock.innerText = `${hours}:${minutes}:${seconds}`;
+}
+getClock();
+setInterval(getClock, 1000);
+function handleAddButtonClick() {
+  const newTodo = toDoInput.value;
+  if (newTodo !== "") {
+    toDoInput.value = "";
+    const newTodoObj = {
+      text: newTodo,
+      id: Date.now(),
+    };
+    toDos.push(newTodoObj);
+    paintToDo(newTodoObj);
+    saveToDos();
+  }
+}
+function saveToDos() {
+  localStorage.setItem("todos", JSON.stringify(toDos));
+}
+if (savedToDos !== null) {
+  const parsedToDos = JSON.parse(savedToDos);
+  toDos = parsedToDos;
+  parsedToDos.forEach(paintToDo);
+}
+function deleteToDo(event) {
+  event.stopPropagation();
+  const li = event.target.parentElement;
+  li.remove();
+  toDos = toDos.filter((toDo) => toDo.id !== parseInt(li.id));
+  saveToDos();
+}
+function paintToDo(newTodo) {
+  const li = document.createElement("li");
+  li.id = newTodo.id;
+  const span = document.createElement("span");
+  span.innerText = newTodo.text;
+  const button = document.createElement("button");
+  button.innerText = "🗑";
+  button.addEventListener("click", deleteToDo);
+  li.addEventListener("click", moveToDone);
+  li.appendChild(span);
+  li.appendChild(button);
```

Comment:
[append()](https://developer.mozilla.org/en-US/docs/Web/API/Element/append) 메서드를 사용하면 여러개의 노드를 한 번에 append 해줄 수 있어서, 한 번 사용해보셔도 좋을 것 같아요! 😄
```suggestion
  li.append(span, button);
```